hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/qjournal/server/TestJournalNode.java

```diff
@@ -280,8 +299,7 @@ public void testJournal() throws Exception {
     assertTrue(lastJournalTimestamp > beginTimestamp);
 
   }
-  
-  
+
```

Comment:
Sorry to bother you, but there's still some change here. Any unrelated change could cause cherry-picking complicated due to the unnecessary whitespace conflicts, that's why I'm suggesting to fix it before merge.

Reply:
Thank you for your review! 

Good suggestion! I have removed unrelated change. Please review it again.